tests/File/GcsFileTest.php

```diff
@@ -49,6 +53,45 @@ public function setUp(): void
         );
     }
 
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testUploadFileWithRefreshedCredentials(): void
```

Comment:
ten test mě padá v SOXu 
![image](https://github.com/keboola/storage-api-php-client/assets/180148/f67fd917-ffda-4ef2-a112-736f1fdb3576)
možeš to zkusit?

Reply:
Skusim, skusal som to viac krat a nemal som problem.

Reply:
Aha on sa ten test este pusta v SOX suite to som si nevsimol, pozriem sa na to. Dik

Reply:
jj tam bude problém že nepoužíváš branch clienta a snažíš sa zapsat do produkce jako developer.

Reply:
<img width="900" alt="Screenshot 2024-01-31 at 15 30 59" src="https://github.com/keboola/storage-api-php-client/assets/6448364/575ec246-2c06-434d-83aa-ab355eeee367">
Dik fixol som to